WordPress/gutenberg
# Fix retrieving autosaves when using a custom rest_namespace

## What?

When specifying `rest_namespace` in a custom post type, a hardcoded `wp/v2` in `getAutosaves` results in a 404

So instead, use the rest namespace from the type, not just the base

## Why?

Because bugs are bad and CPT's with `rest_namespace` deserve autosaves too!

## How?

By not hardcoding `/wp/v2`

## Testing Instructions


1. Register a CPT named `test`
2. In the registration, use `rest_namespace` and set it to something such as `testing/v99`
3. Save permalinks
4. Try to edit a post and there should not be a 404 for `wp/v2/test?autosaves....`  in the browser error console
